rootfs/etc/systemd/system/dietpi-firstboot.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=DietPi-First_boot_installation_process
+DefaultDependencies=false
+After= dietpi-preboot.service
+Before=dietpi-boot.service
+
+[Service]
+Type=oneshot
+StandardOutput=tty
+ExecStartPre=
+ExecStart=/bin/dash -c '/DietPi/dietpi/dietpi-firstboot 2>&1 | tee /var/tmp/dietpi/logs/first_boot_install.log'
```

Comment:
Once run, we could actually remove ```/DietPi/dietpi/dietpi-firstboot``` from system to save 4K+ in RAMdisk, however, would be restored after an ```dietpi-update```.

Would need to add an exclude for ```/DietPi/dietpi/dietpi-firstboot``` in ```dietpi-update```

Reply:
@FredericGuilbault done with https://github.com/FredericGuilbault/DietPi/commit/c0aa1e8cc044b8328f93bd53fffba6515e82769e

Reply:
Saving save 4K+ in RAMdisk is a strong argument. 

But If I delete it. there no way to run it twice and I think re-running the install process is part of @MichaIng dev workflow. So IDK how he feel about that. 

Is it why the partition resize script is in `/var/lib/dietpi/services/fs_partition_resize.sh` and not `/DietPi/dietpi/` ?

Reply:
@FredericGuilbault @Fourdee 
What you think about moving this script to `/var/lib/dietpi/services`? Makes sense to ban code from RAMdisk that only run once.

Reply:
Moving to `/var/lib/dietpi/service` seem good to me. it solve all the needs and make it consistent with `fs_partition_resize.sh` who have the same role.

Reply:
Fixed in 7760a2d